.github/scripts/Get-AlzBicepResourceTypes.ps1

```diff
@@ -0,0 +1,32 @@
+Get-ChildItem -Path '.\infra-as-code\bicep\modules' -Recurse -Filter '*.bicep' -Exclude 'callModuleFromACR.example.bicep', 'orchHubSpoke.bicep' | ForEach-Object {
```

Comment:
Not sure what this script is for if you are doing it as inline script in the workflow?

Reply:
Good spot. I left it here so can be used locally and we could convert the actions at some point in the future. 

Just making it modular and saving future work.